releases/UWP/README.md

```diff
@@ -60,12 +60,12 @@ When using Visual Studio this package requires Visual Studio 2015 Update 3 or la
     - System.Threading.Overlapped
     - System.Xml.XmlDocument
 
-### 5.2.2 (June 27, 2016)
+### 5.2.2 (July 14, 2016)
 When using Visual Studio this package requires Visual Studio 2015 Update 3 or later. You can read more about these changes in the Visual Studio 2015 Update 3 blog [post](https://www.visualstudio.com/en-us/news/releasenotes/vs2015-update3-vs).
```

Comment:
It's incorrect to conflate UWP metapackage version with .NET native version before 5.3.0 (when we have Nuget support). Before that you can mix/match UWP metapackage with any version of .NET native and it should work. It probably makes the most sense to list the .NET Native versions as well as the matching UWP meta package version:

* .NET native 1.0 (UWP MetaPackage 5.1.0 - 5.2.2)
...
* .NET Native 1.4 (UWP MetaPackage 5.1.0 - 5.2.2)
* .NET Native 1.6 (UWP MetaPackage 5.3.0-5.3.2?)
* .NET Native 1.7 (UWP MetaPackage 5.3.3-latest)

Also, you probably want to focus on the .NET native differences. UWP meta package difference in terms of FX should be listed elsewhere. Having said that, given that we are combining the two, it's OK to start listing the FX difference starting from .NET native 1.6. But we should clarify why.